packages/sui-bundler/README.md

```diff
@@ -222,6 +223,51 @@ You could use it to be used offline:
 
 Runtime follows the (API of sw-toolbox)[https://github.com/GoogleChromeLabs/sw-toolbox]. Also, the whitelist is a list of regexp that indicates which pages are secure to use only Client Server Rendering. You could use the `::all::` string to indicate that you always want to use Client Side Rendering.
 
+## Externals Manifest
+If your are using an external CDN to statics assets that are now manage by WebPack, like SVG or IMGs. You can create and `manifest.json` file in the root of your CDN
```

Comment:
`If your are using an external CDN to store statics assets that are now managed by Webpack, like SVG or IMGs, you can create a `manifest.json` file in the root of your CDN (like `https://spa-mock-statics.surge.sh/manifest.json`).